src/backend/aspen/workflows/import_pango_lineages/load_lineages.py

```diff
@@ -90,7 +91,8 @@ def get_lineages(lineage_notes_file: io.TextIOBase) -> list[str]:
     ]
 
 
-def load_lineages_data(lineages: list[str]) -> None:
+# TODO this function is only here for reverse-compatibility. We'll remove it soon
+def load_old_lineages_data(lineages: list[str]) -> None:
```

Comment:
Nit: I think `legacy_load_lineages_data` or `deprecated_load_lineages_data` might be a slightly clearer name, to me the `old` part of this makes it seem like we now have two categories of data, but doesn't make me necessarily think we're going to be dropping support for the old flavor.

Reply:
> Looks good! I don't know that much about databases, but might be worth it to add an index to `pathogen_lineages` DB table on the `pathogen_id` value since whenever we interact with it, we tend to always do stuff that reads/drops all the rows for a given `pathogen_id` at once.

Good call out! The primary key on this table is on [pathogen_id, strain] so it should help us with queries on just pathogen_id. (more info on multi-column indexes here: https://www.postgresql.org/docs/current/indexes-multicolumn.html)

I'll rename the functions in the next pr (which depends on this one)